wat

```diff
@@ -22,6 +26,13 @@ mark_lines() {
 
 : ${PACMAN_LOG:=/var/log/pacman.log}
 
+while [ $# -gt 0 ]; do
+    case $1 in
+        -i|--installed) SHOW_INSTALLED=1; shift ;;
```

Comment:
That seems reasonable. I would initialize it to `0` to make that explicit, up with and following the same syntax as `PACMAN_LOG`.